README.md

```diff
@@ -1 +1,107 @@
-# hybrid-android-example-app
+# Klarna In-App SDK - Hybrid Android Demo App
+![Platforms](https://img.shields.io/badge/platform-android-lightgrey)
+[![GitHub](https://img.shields.io/github/license/klarna/klarna-mobile-sdk)](https://github.com/klarna/klarna-mobile-sdk/blob/master/LICENSE)
+
+This is an example Android app to demonstrate the hybrid integration of [Klarna In-App SDK](https://github.com/klarna/klarna-mobile-sdk).
```

Comment:
Add paragraph describing that the fullscreen transitions have to be handled by the merchant and that what is demonstrated in this app is just one of many ways.